deltachat-ios/AppDelegate.swift

```diff
@@ -568,19 +570,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         let cnt = UserDefaults.standard.integer(forKey: name + "-count")
         UserDefaults.standard.set(cnt + 1, forKey: name + "-count")
         UserDefaults.standard.set(nowTimestamp, forKey: name + "-last")
+    }
 
-        // Values calculated for connectivity view
-        if name == "notify-remote-receive" || name == "notify-local-wakeup" {
-            let timestamps = UserDefaults.standard.array(forKey: Constants.Keys.notificationTimestamps)
-            var slidingTimeframe: [Double]
-            if timestamps != nil, let timestamps = timestamps as? [Double] {
-                slidingTimeframe = timestamps.filter({ nowTimestamp < $0 + 60 * 60 * 24 })
-            } else {
-                slidingTimeframe = [Double]()
-            }
-            slidingTimeframe.append(nowTimestamp)
-            UserDefaults.standard.set(slidingTimeframe, forKey: Constants.Keys.notificationTimestamps)
+    // Values calculated for connectivity view
+    private func increaseDebugDoingFetch() {
```

Comment:
The method name confuses me a little bit, I thought it is actually fetching, but it adding timestamps.

Reply:
happy to rename it - do you have a suggestion?

Reply:
addDebugTimestamp ?

Reply:
addDebugFetchTimestamp